modules/notification/webhook/webhook.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package webhook
+
+import (
+	"code.gitea.io/git"
```

Comment:
`code.gitea.io/git` is an external package?

Reply:
Yes, it is not inside of gitea. IMO